migrate.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
This needs detailed explanation. What is this main() doing? Why does it exist? How does one use it?